Website/Packages/datax-pipeline/src/modules/flowDefinition/components/output/outputSettingsContent.jsx

```diff
@@ -11,6 +11,7 @@ import { Colors, IconButtonStyles, PanelHeader, PanelHeaderButtons, ScrollableCo
 import CosmosDbSinkerSettings from './cosmosdbSinkerSettings';
 import EventHubSinkerSettings from './eventHubSinkerSettings';
 import BlobSinkerSettings from './blobSinkerSettings';
+import SqlSinkerSettings from './SqlSinkerSettings';
```

Comment:
In javascript, try to stick with using lowercase first letter of file names such as sqlSinkerSettings.js; this will also be consistent with all the other files in this codebase.

Reply:
Ah, I see you did case it correctly in the filename below as sqlSinkerSettings.jsx, please update the casing here during the import statement.